content/topics/intro-to-css/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+_db_id: 9
+content_type: topic
+ready: true
+title: Introduction to CSS
+---
+
+> There are two great days in a person's life
+>
+> - the day they are born and the day they discover why.
+
+Topics to be discussed in this workshop can be found here {{% contentlink path="topics/web-frontend/intro-to-css" %}}
```

Comment:
This topic already exists and is added to the recruits backlog so this can be depreciated